setup.py

```diff
@@ -5,6 +5,6 @@
     version="0.0.2",
     python_requires=">= 3.7",
     packages=setuptools.find_packages(include=["mmda.*"]),
-    install_requires=["intervaltree", "pdf2image", "pdfplumber"],
+    install_requires=["intervaltree", "pdf2image", "pdfplumber", "pandas", "requests"],
```

Comment:
`pandas` needed to import `mmda.parsers.pdfplumber_parser`. `requests` needed to call local Docker containers.